File: account/automatic-location-updates/enable-automatic-location-updates-for-devices/code-samples/assignMultipleDevicesAutomaticLocationUpdates.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string accountId = "<ENTER VALUE>";

// POST BODY
AssignMultipleDevicesAutomaticLocationUpdates assignMultipleDevicesAutomaticLocationUpdates = new AssignMultipleDevicesAutomaticLocationUpdates {
    enabledDeviceIds = new[] {
        "<ENTER VALUE>",
    },
    disabledDeviceIds = new[] {
        "<ENTER VALUE>",
    }
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Account(accountId).EmergencyAddressAutoUpdate().Devices().BulkAssign().Post(assignMultipleDevicesAutomaticLocationUpdates);
// PROCESS RESPONSE

File: account/automatic-location-updates/enable-automatic-location-updates-for-devices/code-samples/assignMultipleDevicesAutomaticLocationUpdates.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'enabledDeviceIds' => array(
        '<ENTER VALUE>'  
    ),
    'disabledDeviceIds' => array(
        '<ENTER VALUE>'  
    )
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/account/{$accountId}/emergency-address-auto-update/devices/bulk-assign", $body);
// PROCESS RESPONSE
?>
